src/supertux/menu/options_menu.cpp

```diff
@@ -179,6 +183,65 @@ OptionsMenu::OptionsMenu(bool complete) :
     resolutions.push_back(fullscreen_size_str);
   }
 
+  // Sound Volume
+  sound_volumes.clear();
+  sound_volumes.push_back("0%");
+  sound_volumes.push_back("25%");
+  sound_volumes.push_back("50%");
+  sound_volumes.push_back("75%");
+  sound_volumes.push_back("100%");
+
+  std::ostringstream out;
+  out << g_config->sound_volume << "%";
+  std::string sound_volume = out.str();
+  int cnt_ = 0;
+  for(const auto& volume : sound_volumes)
+  {
+      if(volume == sound_volume)
+      {
+          sound_volume.clear();
+          next_sound_volume = cnt_;
+          break;
+      }
+      ++cnt_;
+  }
+
+  if (!sound_volume.empty())
+  {
+      next_sound_volume = sound_volumes.size();
+      sound_volumes.push_back(sound_volume);
```

Comment:
Actually I don't think that if makes sense at all. I'll change to something different.